`llvm-kompile` crashes with bad error when #Ceil rule is added

I'm not sure if I'm using the `#Ceil` rule correctly, but I've marked it `symbolic` and still the LLVM backend is crashing with an uninformative error.

At the very least, the error should be made more informative about the problem. But I also think:

-   Marking it `symbolic` should mean the LLVM backend ignores it.
-   We should be able to send `#Ceil` rules to the LLVM backend and have it ignore them.

-   K commit: 4a011c920 - haskell-backend/src/main/native/haskell-backend: b3c17111 - Fix sorting bugs around RulePattern (kframework/kore#1787) (kframework/k#1251)
-   LLVM backend commit: fa4c44f - fix non-exhaustiveness bug (#324)

K Definition:

```
module FIXED-INT 
    imports INT 
    imports BOOL 
 
    syntax FInt ::= FInt ( value: Int , one: Int ) [klabel(FInt), symbol] 
 // --------------------------------------------------------------------- 
    rule #Ceil(FInt(... value: V, one: O)) => { V >=Int 0 andBool O >Int 0 #Equals true } [anywhere, symbolic] 
endmodule 
```

Error output:

```
/home/dev/src/mkr-mcd-spec-dev/deps/k/k-distribution/target/release/k/lib/../bin/llvm-kompile: line 44: 59186 Segmentation fault      (core dumped) "$(dirname "$0")"/llvm-kompile-codegen "$definition" "$dt_dir"/dt.yaml "$dt_dir" $debug > "$mod"
org.kframework.utils.errorsystem.KEMException: [Error] Critical: llvm-kompile returned nonzero exit code: 139
Examine output to see errors.
        at org.kframework.utils.errorsystem.KEMException.create(KEMException.java:126)
        at org.kframework.utils.errorsystem.KEMException.criticalError(KEMException.java:39)
        at org.kframework.backend.llvm.LLVMBackend.accept(LLVMBackend.java:76)
        at org.kframework.kompile.KompileFrontEnd.run(KompileFrontEnd.java:74)
        at org.kframework.main.FrontEnd.main(FrontEnd.java:62)
        at org.kframework.main.Main.runApplication(Main.java:118)
        at org.kframework.main.Main.runApplication(Main.java:108)
        at org.kframework.main.Main.main(Main.java:56)
Makefile:104: recipe for target '.build/defn/llvm/fixed-int-kompiled/interpreter' failed
make: *** [.build/defn/llvm/fixed-int-kompiled/interpreter] Error 113
```

Kompile command:

```
kompile --debug --main-module FIXED-INT --backend llvm              \
                 --syntax-module FIXED-INT .build/defn/llvm/fixed-int.k \
                 --directory .build/defn/llvm -I .build/defn/llvm                           \
                 --emit-json                                                      \
                  -ccopt -O2
```

--- a/k-distribution/tests/regression-new/checks/anywhereSymbolic.k
+++ b/k-distribution/tests/regression-new/checks/anywhereSymbolic.k
@@ -1,0 +1,7 @@
+module ANYWHERESYMBOLIC
+
+  syntax Foo  ::= a() | b()
+
+  rule a() => b() [anywhere, symbolic]
+
+endmodule

--- a/k-distribution/tests/regression-new/checks/anywhereSymbolic.k.out
+++ b/k-distribution/tests/regression-new/checks/anywhereSymbolic.k.out
@@ -1,0 +1,6 @@
+[Error] Compiler: anywhere attribute is not supported on symbolic rules.
+	Source(anywhereSymbolic.k)
+	Location(5,8,5,18)
+	5 |	  rule a() => b() [anywhere, symbolic]
+	  .	       ^~~~~~~~~~
+[Error] Compiler: Had 1 structural errors.

--- a/kernel/src/main/java/org/kframework/compile/checks/CheckAtt.java
+++ b/kernel/src/main/java/org/kframework/compile/checks/CheckAtt.java
@@ -185,5 +185,8 @@ public class CheckAtt {
         if(att.contains(Att.ANYWHERE()) && att.contains(Att.SIMPLIFICATION())) {
           errors.add(KEMException.compilerError("anywhere attribute is not supported on simplification rules.", loc));
         }
+        if(att.contains(Att.ANYWHERE()) && att.contains(Att.SYMBOLIC())) {
+            errors.add(KEMException.compilerError("anywhere attribute is not supported on symbolic rules.", loc));
+        }
     }
 }

